.travis-ci.sh

```diff
@@ -1,5 +1,5 @@
 # Edit this for your own project dependencies
-OPAM_DEPENDS="ocamlfind extlib ulex"
+OPAM_DEPENDS="ocamlfind sedlex extlib dune dune-build-info"
```

Comment:
I think you said now we use custom library instead of sedlex, am I right?
Then `sedlex` part is unnecessary, isn't it?

Reply:
`sedlex` is the library for the _lexer_. The _parser_ is using the custom parser combinators.